client/src/components/molecules/NotificationPopup.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <q-btn
+    flat
+    padding="none"
+    data-cy="dropdown_notificiations"
+    class="notifications-dropdown"
+    :aria-label="$t('header.notification_button')"
+  >
```

Comment:
I noticed this introduces an "Interactive controls must not be nested" error in axe DevTools. How should we proceed?